wallet-test/src/test/scala/org/bitcoins/wallet/dlc/DLCNegotiationTest.scala

```diff
@@ -0,0 +1,101 @@
+package org.bitcoins.wallet.dlc
+
+import org.bitcoins.dlc.DLCMessage
+import org.bitcoins.testkit.wallet.FundWalletUtil.FundedWallet
+import org.bitcoins.testkit.wallet.{BitcoinSWalletTest, DLCWalletUtil}
+import org.scalatest.FutureOutcome
+
+class DLCNegotiationTest extends BitcoinSWalletTest {
+  type FixtureParam = FundedWallet
+
+  override def withFixture(test: OneArgAsyncTest): FutureOutcome = {
+    withFundedSegwitWallet(test)
+  }
+
+  behavior of "DLCNegotiation"
+
+  it must "create a dlc offer" in { funded: FundedWallet =>
+    val fundedWallet = funded.wallet
+
+    val offerData = DLCWalletUtil.sampleDLCOffer
+
+    for {
+      offer <- fundedWallet.createDLCOffer(
```

Comment:
might be worthwhile to create an overloaded version of `createDLCOffer` that takes in a `DLCOffer`